tractdb_pyramid/views/storytelling_chapterview.py

```diff
@@ -6,8 +6,8 @@
 
 service_chapters = cornice.Service(
     name='storytelling_chapters',
-    path='/storytelling/chapters',
-    description='Get all chapters associated with the logged in user',
+    path='/storytelling/chapters/{story_id}',
```

Comment:
This is wrong (i.e., not RESTful). Should be `/storytelling/{story_id}/chapters`.

You're asking for the chapters on a given story.  Think of it as a hierarchy.